src/index.js

```diff
@@ -1,35 +1,14 @@
 ﻿import GpsApiLocationSource from "./locationSource/GpsApiLocationSource";
-import WeatherAPI from './weatherSource/WeatherSource'
-// plik inicjalizacyjny aplikacji
+import OpenWeatherMapApi from './weatherSource/WeatherSource'
 
-console.log("...");
+var WeatherAPI = new OpenWeatherMapApi('http://api.openweathermap.org/data/2.5/','953349aaa2569f9cd4821f8c2ffda23a')
```

Comment:
To są zasadniczo bazgroły testowe w index.js, ale i taka luźna uwaga, że nie jest dobrze nazywać zmienne z wielkiej litery ;)